File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        // I need a fking counter array to count elements but first I check if size is equal or no
        if(s.size()!=t.size()){
            return false;
        }
        vector<int> query(26,0);
        for(int i= 0; i<s.size();i++){
            query[s[i]-'a']+=1;
            query[t[i]-'a']-=1;
        }
        for(int i : query){
            if(i!=0) return false;
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s)!= len(t):
            return False
        query = [0]*26
        for cs,ct in zip(s,t):
            query[ord(cs)-ord('a')]+=1
            query[ord(ct)-ord('a')]-=1
        for i in query:
            if i!=0:
                return False
        return True